reasoner_util/__init__.py

```diff
@@ -38,16 +38,22 @@ def normalize_ids(curies: List[str]) -> List[str]:
     return normalized_ids
 
 
-def normalize_qcategories(catagories: List[str]) -> List[str]:
-    """Normalize a list of catagories by stripping all descendents"""
-    normalized_catagories = strip_descendants(catagories)
-    return normalized_catagories
+def normalize_qcategories(message_dict: dict) -> None:
+    """Normalize categories for each node in the query graph by stripping
+    all descendants. Modify the message in place."""
+    q_nodes = message_dict["message"]["query_graph"]["nodes"]
+    for q_node in q_nodes.values():
+        q_node["categories"] = strip_descendants(q_node["categories"])
+    return
```

Comment:
This `return` is unnecessary since the function will automatically return `None` when it reaches the bottom of the block.